index.html

```diff
@@ -8,12 +8,10 @@
     <meta name="MobileOptimized" content="320">
     <meta http-equiv="cleartype" content="on">
     <style>
-      html,
-      body,
       canvas {
-        margin: 0 !important;
-        padding: 0 !important;
-        overflow: hidden !important;
+        width: 100%;
+        max-width: 1200px;
+        max-height: 100vh;
```

Comment:
This probably isn't the final form here, but added these to make re-sizing the window a bit more pleasant.